src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -68,30 +68,8 @@ public static Command parse(String fullCommand) throws DukeException {
 
 
         case ListCommand.COMMAND_WORD:
-            if (fullCommand.equals("list")) { //list every tasks
-                return new ListCommand();
-            }
-            int priority;
-            String category;
-            if (commandString.contains("p/")) {
-                if (argumentsMap.get("p") == null) {
-                    throw new DukeException(Messages.EXCEPTION_EMPTY_PRIORITY);
-                }
-                try {
-                    priority = Integer.parseInt(argumentsMap.get("p"));
-                } catch (NumberFormatException e) {
-                    throw new DukeException(Messages.EXCEPTION_INVALID_INDEX);
-                }
-                return new ListCommand(priority);
-            } else if (commandString.contains("c/")) {
-                if (argumentsMap.get("c") == null) {
-                    throw new DukeException(Messages.EXCEPTION_EMPTY_CATEGORY);
-                }
-                category = argumentsMap.get("c");
-                return new ListCommand(category);
-            } else {
-                throw new DukeException(Messages.EXCEPTION_INVALID_LIST_COMMAND);
-            }
+            //checkAllowedArguments(argumentsMap, ListCommand.ALLOWED_ARGUMENTS);
```

Comment:
I only used this line to check if the user entered valid arguments but can be removed for your part of the code if needed.